Makefile

```diff
@@ -69,20 +72,18 @@ ticks:
 	$(MAKE) -C src/ticks
 	$(MAKE) -C src/ticks PREFIX=`pwd` install
 
-config:
-	cp -f lib/config/$(DEFAULT).cfg lib/config/zcc.cfg
 
 libs:
 	cd libsrc ; $(MAKE)
 	cd libsrc ; $(MAKE) install
 
 install-libs:
-	mkdir -p $(prefix)/lib/z88dk/lib/config
-	cp -R lib/config/* $(prefix)/lib/z88dk/lib/config/
-	find $(prefix)/lib/z88dk/lib/config -type f | xargs chmod 644
-	mkdir -p $(prefix)/lib/z88dk/lib/clibs
-	cp -R lib/clibs/* $(prefix)/lib/z88dk/lib/clibs/
-	find $(prefix)/lib/z88dk/lib/clibs -type f | xargs chmod 644
+	mkdir -p $(DESTDIR)/$(prefix_share)/z88dk/lib/config
+	cp -R lib/config/* $(DESTDIR)/$(prefix_share)/z88dk/lib/config/
```

Comment:
Strictly speaking, you'd use $INSTALL, but if there's lots of sub-dirs, could be annoying.